grr/server/grr_response_server/gui/api_plugins/timeline.py

```diff
@@ -152,15 +158,22 @@ def _GenerateTimeline(
       client_id: Text,
       flow_id: Text,
       fqdn: Text,
+      format: ApiGetCollectedTimelineArgs.Format,
       zipgen: utils.StreamingZipGenerator,
   ) -> Iterator[bytes]:
     args = ApiGetCollectedTimelineArgs()
     args.client_id = client_id
     args.flow_id = flow_id
-    # TODO(hanuszczak): Add support for other formats.
-    args.format = ApiGetCollectedTimelineArgs.Format.RAW_GZCHUNKED  # pytype: disable=attribute-error
+    args.format = format
+
+    if format == ApiGetCollectedTimelineArgs.Format.RAW_GZCHUNKED:
+      filename = f"{client_id}_{fqdn}.gzchunked"
+    elif format == ApiGetCollectedTimelineArgs.Format.BODY:  # pytype: disable=attribute-error
```

Comment:
Interesting, why `pytype` directive? We don't have any `pytype` in our open source tree, do we? Also, why is it needed here but not two lines above?

Reply:
Oups! I reused this line as it was from the previous version of the file, but forgot about the comment. Thanks!